action.yml

```diff
@@ -37,9 +42,16 @@ runs:
 
     - name: Initialise LXD
       shell: bash
+      env:
+        # Use an environment variable to have correct quoting and escaping
+        INPUT_PRESEED: ${{ inputs.preseed }}
       run: |
         sudo lxd waitready
-        sudo lxd init --auto
+        if [[ "${INPUT_PRESEED}" == "" ]]; then
+          sudo lxd init --auto
```

Comment:
I'd probably move the `images.compression_algorithm` tweak here so that preseed users could alter it.

It might be useful to update the test preseed to include it assuming that speeds up CI.

Reply:
Makes sense, I have changed it.